www/cli.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
This file should be in a different folder - the www folder should contain the assets that are available & required for browsers - cli folder doesn't belong here